src/pages/_app.page.tsx

```diff
@@ -6,6 +6,86 @@ import { NetworkProvider } from 'providers/NetworkProvider'
 import '../styles/antd.css'
 import '../styles/index.scss'
 
+<<<<<<< HEAD
```

Comment:
My editor shipped with this new thing, https://code.visualstudio.com/updates/v1_69#_3-way-merge-editor.

I thought it resolved conflicts, but I guess not. Will update this PR's conflicts

Reply:
Will create a new PR